### PR TITLE
Do not start service on a new install. Allow users to configure the agent and server and have them start it up. (#3119)

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

**Comment by ketan** [comment]
Here's the diff of the package scripts on debian —

https://gist.github.com/ketan/460271873845ad46d897015f15b48dbe


Here's the diff of the package scripts on RPMS —

https://gist.github.com/ketan/80cd68bc1eea627d80c0eea877fe8bfc/revisions

File: server/config/cruise-config.xml [hunk 1]
```diff
@@ -1,28 +1,80 @@
 <?xml version="1.0" encoding="utf-8"?>
-<cruise xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="cruise-config.xsd" schemaVersion="87">
+<cruise xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="cruise-config.xsd" schemaVersion="88">
```
Comment (varshavaradarajan) [inline server/config/cruise-config.xml]: This file shouldn't be checked in, right?

Comment (ketan) [inline server/config/cruise-config.xml]: Probably not as part of this PR. But if we bump the version, we should just check this file in.

Comment (varshavaradarajan) [inline server/config/cruise-config.xml]: Sure. Its already done though. https://github.com/gocd/gocd/blob/master/server/config/cruise-config.xml.

**Comment by varshavaradarajan** [comment]
Pending: Show a post install message on fresh install of the go server.

**Pull request merged**